config/default/simpleExporter.conf.php

```diff
@@ -68,5 +72,19 @@
                 'valuesAsColumns' => true,
             )
         ),
+        'metadataProperties' => array(
+            'extractor' => 'MetaDataOntologyExtractor',
+            'parameters' => array(
+                'valuesAsColumns' => true,
+                'excludedProperties' => array(
+                    taoItems_models_classes_ItemsService::PROPERTY_ITEM_CONTENT,
+                    taoItems_models_classes_ItemsService::PROPERTY_ITEM_MODEL,
+                    taoTests_models_classes_TestsService::PROPERTY_TEST_TESTMODEL,
+                    taoTests_models_classes_TestsService::TEST_TESTCONTENT_PROP,
```

Comment:
taoTests isn't required by taoQtiItem (it is the other way, Tests requires Items)
This exclude property should be in the taoQtiTest metadata exporter

Reply:
DONE